db/memorystore.go

```diff
@@ -16,6 +16,17 @@ import (
 	"github.com/letsencrypt/pebble/core"
 )
 
+// Error when key change fails because new account key has already been in use
```

Comment:
The Godoc convention is to start a type's comment with the name of the type, like:
> // ExistingAccountError is an error type indicating when an operation fails because the `MatchingAccount` has a key conflict.

Reply:
Done.